### ISSUE TITLE
carthage is fetching RxSwift all the time

### ISSUE THREAD

**Issue by RiuHDuo** [Moya/Moya#1474]
carthage is fetching all the time
how to solve this？
i use carthage 0.26.2

**Comment by RiuHDuo** [comment]
i find that problem. I write github "Moya/Moya" in carthage . i check out moya 7.0.4. Why dont check out 10.0?

**Comment by SD10** [comment]
Hey @RiuHDuo,
I know Moya 10.0 uses ReactiveSwift 2.0.0. Could this possibly be the issue?

**Comment by RiuHDuo** [comment]
i don't wanna use any moya rx or reactive extension. i just use stander moya framework.

**Comment by lucas34** [comment]
Moya's Cartfile specify dependencies to RxSwift and ReactiveSwift. Regardless you need it or not. I don't know if Carthage allows you to specify multiple Cartfiles.

**Comment by RiuHDuo** [comment]
I hope you should make 2 projects. One is stander Moya and the other is Rx/ReactiveCocoa Extension like PromiseKit and PromiseKit Extensions. And i will not be clone some project i never use like Rx series frameworks or Reactive series frameworks

**Comment by pedrovereza** [comment]
@RiuHDuo The discussion about splitting the project into multiple repos comes back every once in a while. See https://github.com/Moya/Moya/issues/345 and https://github.com/Moya/Moya/issues/1125.

**tl;dr:**
* The problem is on Carthage side;
* Spliting into multiple repos makes it easier for users, but adds a lot of complexity on our side

**Comment by RiuHDuo** [comment]
Finally, i find the solution.
Set the version of every framework in Cartfile.resolved file manually.
And run 
```
  carthage bootstrap
```
